packages/core/src/VDocument.ts

```diff
@@ -1,10 +1,11 @@
 import { VNode } from './VNodes/VNode';
 import { VRange } from './VRange';
-import { CharNode, FormatType, FORMAT_TYPES } from './VNodes/CharNode';
+import { CharNode } from '../../plugin-char/src/VNodes/CharNode';
```

Comment:
WIP! Nothing in this file is expected to work for the moment. The main question is how to make this plugin-agnostic.
For example I thought of using `execCommand` to let `Char` create the `CharNodes` but `execCommand` doesn't return anything.